api/config/config_local.json

```diff
@@ -125,5 +125,19 @@
     "dataset": "reporting_local",
     "maxRowsPerInsert": 200,
     "uploadMethod": "STREAMING"
+  },
+  "accessTiers": {
```

Comment:
I'm slightly concerned about having these config objects in two separate places, since they are pushed at different times, and are read from the db at slightly different timings. Here's an alternative idea I had, let me know what you think:

Update cdr_versions_*.json as follows:
```
{
  accessTiers: {
    "registered": {},
    "prototype": {}
  },
  cdrVersions: [
    {...},
    ...
  ]
}
```

This would allow the config to be self-consistent, and `UpdateCdrVersions` could check that the config file is at least self-consistent. This would probably also require introducing a table for `accessTiers`, which may or may not be desirable.

Reply:
I see your point but it feels awkward to me have tier definitions live in a file named "cdr versions".

Is your concern reduced if a tier update is a very rare occurrence?  I expect it would be.  We may never need to do so after Controlled Tier is live.

Reply:
A tier should be able to contain zeror, one, or many CDR versions, ideally even allowing different CDR Flavors/data assets. No one has asked for this, but stakeholders will assume we can support it so we might as well prepare.

Reply:
Spelling out specific concerns:
- Config management overhead of having these dependent concepts split across two files
- Release management overhead when introducing a new tier in an environment or changing the tier metadata (changing the details probably won't be common, but adding a column, removing a column etc are likely scenarios). config and cdr_versions are not pushed transactionally together, so we need to consider which one goes first, which one rolls back first, how to stage a change across both, etc
- UpdateCdrVersions tool will have difficulty validating a change if the `config` details are out of scope. We can add consistency checks and apply the change transactionally from a single tool if this tool owns pushing/updating both things.

These are not major concerns, but all else being equal, sticking these things together may be easier to manage.

RE filename: I'd probably just rename it to something more general like `cdr_config_*.json` if we went with this approach

Reply:
#4510 addresses these concerns for the post-prototype implementation